src/Components/Btn.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Btn extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: props.value,
+    };
+  }
+
+  static getClassName(value) {
+    const classNames = {
+      AC: 'btn1',
+      '+/-': 'btn1',
+      '%': 'btn1',
+      '÷': 'btn2',
+      x: 'btn2',
+      '-': 'btn2',
+      '+': 'btn2',
+      '=': 'btn2',
+      0: 'zero',
+      '.': 'btn1',
+    };
+    return classNames[value] || 'num';
+  }
+
+  render() {
+    const { value } = this.state;
+    return (
+      <button type="button" className={`${Btn.getClassName(value)} btn`}>
+        {value}
+      </button>
+    );
+  }
+}
+
+Btn.propTypes = {
+  value: PropTypes.string.isRequired,
+};
+
+export default Btn;
```

Comment:
- [ ] One of the requirements is that you should use _**functional**_ components instead of _**class-based**_ components. Kindly implement this requirement for this component. 👍 😎